enterprise/frontend/src/metabase-enterprise/sandboxes/components/MappingEditor.tsx

```diff
@@ -2,22 +2,54 @@
 import _ from "underscore";
 import { t } from "ttag";
 
-import Button from "metabase/core/components/Button";
+import type React from "react";
+import { Button, TextInput } from "metabase/ui";
```

Comment:
Migrated this to TS and `metabase/ui` components, functionality should not have changed